src/assets/sass/themes/button.scss

```diff
@@ -0,0 +1,104 @@
+@import "src/assets/sass/variables/variables";
+
+.btn {
+  display: inline-block;
+  font-weight: 400;
+  color: #212529;
+  text-align: center;
+  vertical-align: middle;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  background-color: transparent;
```

Comment:
better use of defined transparency (just an oppinion)

Reply:
What is your suggestion?